topbeat/beater/topbeat.go

```diff
@@ -49,9 +47,9 @@ func (tb *Topbeat) Config(b *beat.Beat) error {
 		tb.period = 10 * time.Second
 	}
 	if tb.TbConfig.Input.Procs != nil {
-		tb.procs = *tb.TbConfig.Input.Procs
+		tb.procStats.Procs = *tb.TbConfig.Input.Procs
```

Comment:
I wouldn't place `procs` under `procStats` as I think it is a bit confusing.  `procStats` is just the configuration option that if enabled then statistics about the processes are exported, so I would see it more top level. I think `procStats` is not a good name and we should change it with something more obvious like `exportProcStats`.